lib/components/post-editor-sidebar-component.js

```diff
@@ -90,7 +90,7 @@ export default class PostEditorSidebarComponent extends BaseContainer {
 	}
 
 	publicizeToTwitterAccountDisplayed() {
-		const twitterAccountSelector = By.xpath( `//h5[text()='Twitter']/..//label` );
+		const twitterAccountSelector = By.css( 'div.editor-sharing__publicize-connection > label > span' );
```

Comment:
Thanks, can you please create a pull request for your Calypso change so we can get it reviewed and merged in so that these changes can take effect?